ts/Dashboards/EditMode/Sidebar.ts

```diff
@@ -838,22 +840,27 @@ class Sidebar {
                     type = componentSettings[key].type;
 
                     if (key === 'chartType') {
-                        const chartTypesEnum: Array<string> = [
+                        const chartTypesEnum = [
                             'column',
                             'line',
                             'scatter',
                             'spline',
                             'pie'
                         ];
 
-                        const chartOpts =
-                            (currentComponent as any).chartOptions;
+                        // eslint-disable-next-line
```

Comment:
How about instead of casting it to a certain type, check it like this.
```TS
                        if (currentComponent instanceof HighchartsComponent) {
                            ... your code
                        }
```

Reply:
If this is done, we have Highcharts in the Dashboard bundle. Maybe this should become an event somehow and HighchartsComponent should listen to the Sidebar to do whatever is necessary.

Reply:
Events in the sidebar sound good, but I would like to refactor the sidebar in the next step when we will design all controllers inside.

Any suggestions which solution is better: use `instanceof` or left like it is.

Reply:
For me it is fine as it is a temporary thing.